hadoop-ozone/dist/src/main/smoketest/omha/om-roles.robot

```diff
@@ -19,12 +19,37 @@ Resource            ../commonlib.robot
 Test Timeout        5 minutes
 Test Setup          Run Keyword if    '${SECURITY_ENABLED}' == 'true'    Kinit test user     testuser     testuser.keytab
 
+*** Keywords ***
+Parse non json output
```

Comment:
Nit: I think `Assert Leader Present` or `Output Should Contain Leader` would better describe this keyword.

Reply:
Done